java/repo.bzl

```diff
@@ -0,0 +1,21 @@
+load("//java/third_party:workspace.bzl", "maven_dependencies")
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_jar")
+
+def __maybe(repo_rule, name, **kwargs):
+    if name not in native.existing_rules():
+        repo_rule(name = name, **kwargs)
+
+def __maven_repositories():
+    maven_dependencies()
+
+def __bazel_deps():
```

Comment:
again I'd remove the underscores (if the name collides, we can name it something else)